deploy/Vagrantfile

```diff
@@ -88,4 +89,46 @@ Vagrant.configure(2) do |config|
 
     adterm_config.vm.network :private_network, ip: "10.0.1.2"
   end
+
+  # Kerberos Server VM - for testing purposes only
+  config.vm.define :kerberos_server do |kerberos_config|
+    kerberos_config.vm.box = "chef/centos-6.5"
+    kerberos_config.vm.host_name = "kdc.test.dev"
+
+    kerberos_config.vm.provision :shell,
+      :inline => "sudo echo '10.0.1.3  proxy.test.dev' >> /etc/hosts"
+
+    # For getting around an issue in Virtualbox/Vagrant
+    # kerberos_config.vm.provision :shell,
+    #   :inline => "sudo yum install kernel-devel-2.6.32-431.el6.x86_64"
+
+    kerberos_config.vm.provision "ansible" do |ansible|
+      ansible.playbook = "deploy_kerberos_server.yml"
+    end
+
+    kerberos_config.vm.network :private_network, ip: "10.0.1.127"
+  end
+
+  # Enclave Proxy VM
+  config.vm.define :proxy_server do |proxy_config|
+    proxy_config.vm.box = "chef/centos-6.5"
+    proxy_config.vm.host_name = "proxy.test.dev"
+
+    proxy_config.vm.provision :shell,
+      :inline => "sudo echo '10.0.1.127  kdc.test.dev' >> /etc/hosts"
+
+    proxy_config.vm.provision :shell,
+      :inline => "sudo echo '10.0.1.3  proxy.test.dev' >> /etc/hosts"
+
+    proxy_config.vm.provision "ansible" do |ansible|
+      ansible.playbook = "deploy_proxy_server.yml"
+
+      ansible.groups = {
+        "proxy_server" => ["proxy_server"],
+        "all_groups:children" => ["proxy_server"]
+      }
+    end
+
```

Comment:
We can, although currently you can view the proxy server just by going to port 80 on 10.0.1.3 from the host.  Do we want to change ports for another reason?